docs/modules/language-tutorial/pages/02_filling_out_a_template.adoc

```diff
@@ -58,11 +58,11 @@ parrot {
 }
 ----
 
-[IMPORTANT]
+[NOTE]
 ====
-_Amending_ does not allow us to _add_ properties to the (typed) object we are amending.
+So far, we have only worked with link:{uri-stdlib-Dynamic}[Dynamic] objects. The other type available to us is link:{uri-stdlib-Typed}[Typed] objects. _Amending_ allows us to override, amend, and add new properties to a dynamic object, but a typed object will only let us amend or override existing properties, not add entirely new ones.
```

Comment:
One code style note: We keep to one sentence per line (gives somewhat nicer diffing over time).

For presentation purposes, we prefer the _second_ person (it's more activating).

Although I agree with this fix, I feel some wordsmithing is called for, especially because `Listing` and `Mapping` live at the same level as `Dynamic` and `Typed`. Leaving it to you to accept/alter:
```suggestion
So far, you have only worked with link:{uri-stdlib-Dynamic}[Dynamic] objects.
Pkl also offers link:{uri-stdlib-Typed}[Typed] objects.
_Amending_ allows you to override, amend, and add new properties to a dynamic object.
Typed objects will only let you amend or override existing properties, not add entirely new ones.
```

Reply:
Thanks, that's an improvement; applied 🙏🏻